CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,5 @@
 - Fix: threadpool mode not actually used for custom notifications, even if -notificationMode is set so (#2622)
 - Fix: -logAppend as default in configuration for RPM package (#2637)
+- Fix: Default log output for stdout is now the same format as in the log file. Use new CLI option '-logForHumans' to nodify this. (Issue #2609)
```

Comment:
Line 2 should be removed. Line 3 is probably redundant as -logForHumans is already mentioned in line 1.